charts/quantum-serverless/values.yaml

```diff
@@ -53,51 +43,6 @@ gateway:
         clientSecret: GATEWAYSECRET-CHANGEME
     superuser:
       enable: true
-    service:
-      type: ClusterIP
-      port: 8000
-    ray:
-      nodeImage: "icr.io/quantum-public/quantum-serverless-ray-node:0.6.0-py39"
-      opensslImage: registry.access.redhat.com/ubi8/openssl:8.8-7
-      kubectlImage: alpine/k8s:1.28.1@sha256:c37a54b738e4d26f88d4a4d28680bb74dfea1d0e756f039dad900306abc837a8
-    limits:
-      maxJobsPerUser: 2
-      maxComputeResources: 4
-  cos:
-    claimName: gateway-claim
-
-  secrets:
```

Comment:
I think you will need to promote this change to `gateway/values.yaml`.

Reply:
Yes, they are already there except `service`.  The `service` here is in the wrong place.